components/kyma-environment-broker/internal/orchestration/manager/manager.go

```diff
@@ -218,6 +256,16 @@ func (m *orchestrationManager) waitForCompletion(o *internal.Orchestration, stra
 		if found {
 			numberOfNotFinished += numberOfPending
 		}
+		numberOfRetrying, found := stats[orchestration.Retrying]
+		if found {
+			numberOfNotFinished += numberOfRetrying
+			err = m.factory.ConvertRetryingToPendingOperations(o.OrchestrationID)
```

Comment:
Before converting them, you would need to look them up, and signal `strategy.Retry(ops)` so that they are reprocessed by the current orchestration stretegy.
This is not yet implemented.

Reply:
thanks for pointing this out.. added inserting the ops to the delaying queue after the converting